src/components/widgets/camera/CameraItem.vue

```diff
@@ -77,16 +70,33 @@ export default class CameraItem extends Vue {
   // Maintains the last cachebust string
   refresh = new Date().getTime()
 
+  // Callback to cancel requestAnimationFrame() when component is being destroyed.
+  cancelCameraTransform = noop
+
   /**
    * Handle any transforms the user may have set on the camera image.
    */
-  get cameraTransformStyle () {
+  cameraTransformStyle (element: HTMLElement) {
     const config = this.camera
     let transforms = ''
-    transforms += (config && config.flipX) ? ' scaleX(-1)' : ''
-    transforms += (config && config.flipY) ? ' scaleY(-1)' : ''
-    // transforms += ` rotate(${config.rotate}deg)`
-    return (transforms.trimLeft().length) ? { transform: transforms.trimLeft() } : {}
+
+    if (!config.rotate) {
+      transforms += config && config.flipX ? ' scaleX(-1)' : ''
+      transforms += config && config.flipY ? ' scaleY(-1)' : ''
+    } else {
+      let scaling = element.clientHeight / element.clientWidth
```

Comment:
the scaling works fine for 90° and 270° rotation, though i am not sure about the black bars (they are probably not easily removable because of the footer text tho; and maybe even fine considering each camera view is meant to split the card evenly):
![90 deg rotation](https://user-images.githubusercontent.com/25269274/151714784-b92eef97-6ffb-42bf-8e6f-2a06cced37cb.png)

for 180° (and 360°) the scaling is quite off:
![180 deg rotation](https://user-images.githubusercontent.com/25269274/151714812-28a44ec8-536b-4fab-adc8-b8928ae6e41f.png)

Reply:
Yes, there are some questionable cases that would require scaling the parent elements, so I didn't intend to change too much. Maybe I'll try it for the next iteration.

My camera is currently not usable on my printer, but I'll check this presumably Monday/Tuesday evening. I also tried it with my 1080p camera, which worked on a maximized browser window. Can you leave your browser resolution and just in case the camera resolution? I don't see a reason it would be different as long as the aspect ratio stays the same, but oh well.

Reply:
I'm using Chrome on Win10 on a 1080p monitor with a 1080p camera. I assume scaling at 180° can be omitted completely, because it will just result in the same canvas size.